packages/frontend-2/components/viewer/resources/ActiveVersionCard.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="bg-primary-muted flex gap-2 py-2 px-4">
+    <EyeIcon v-tippy="'Currently viewing'" class="h-4 w-4 text-primary mt-0_5" />
```

Comment:
I noticed that this icon is focusable and gets a black outline on focus, please remove it

Reply:
Also, if the version message is very long, the icon becomes very small. You probably need to set `flex-shrink: 0` onto it.